zuul.d/jobs.yaml

```diff
@@ -157,8 +138,6 @@
     description: |
```

Comment:
this designate job use the specific run.yaml, don't use its parents' script, so I think we can set parent as "golang-test".

Reply:
both 'golang-test' and 'terraform-provider-openstack-acceptance-test' as parent job will have no impact on the designate job execution however, I agree with you as choosing the 'golang-test' as parent job is more  logical.